### ISSUE TITLE
Serious defect with Multiple Participants on an Event not being charged on PayPal

### ISSUE THREAD

**Issue by lsmithgo** [eileenmcnaughton/nz.co.fuzion.omnipaymultiprocessor#127]
Eileen

As in the title, if one registers more than one participant for an event, the charge made via PayPal checkout is only for the first participant.  However, all the other participants are registered, contributions created and marked as paid.

**Comment by eileenmcnaughton** [comment]
I've dug into this a little - it's pretty tricky - the fix will probably have to be a core fix

**Comment by lsmithgo** [comment]
Yes, it would seem to need a change to the Event booking workflow to put the PayPal Checkout at the end of the process.  In fact, such a restructuring would be good for all multiple participant bookings as it is odd today that the payment details are taken before all the participant details have been collated.

**Comment by andyanderso** [comment]
I am also experiencing this issue. Is there a work-around or any update on a potential fix?
